build/build.sh

```diff
@@ -4,12 +4,15 @@ set -ex
 
 VERSION=$1
 OS=linux
+TIMESTAMP="$(date +%Y%m%d)"
```

Comment:
Can you set `CHECKED_BUILD_NEEDED=0` here - I'm surprised this isn't erroring on line 45 with the `set -u`

Reply:
oh! there is no `set -u` hmm. ok I guess this OK compared to the rest of the file but it's something to worry about another time.